samples/ingestion/ingestion-client/FetchTranscription/Language/AnalyzeConversationsProvider.cs

```diff
@@ -0,0 +1,255 @@
+                if (piiErrors.Any())
+                {
+                    errors.AddRange(piiErrors.SelectMany(s => s.Errors).Select(s => $"Error thrown for conversation : {s.Id}"));
+                }
+            }
+
+            return (piiResults, errors);
+        }
+
+        /// <summary>
+        /// Checks for all conversational analytics requests that were marked as running if they have completed and sets a new state accordingly.
+        /// </summary>
+        /// <param name="conversationRequests">Enumerable for conversationRequests.</param>
+        /// <returns>True if all requests completed, else false.</returns>
+        public async Task<bool> ConversationalRequestsCompleted(IEnumerable<Connector.Serializable.TranscriptionStartedServiceBusMessage.TextAnalyticsRequest> conversationRequests)
+        {
+            if (!IsConversationalPiiEnabled())
+            {
+                return true;
+            }
+
+            if (conversationRequests == null || !conversationRequests.Any())
+            {
+                return true;
+            }
+
+            var runningJobsCount = 0;
+
+            foreach (var textAnalyticsJob in conversationRequests)
+            {
+                var response = await ConversationAnalysisClient.GetAnalyzeConversationJobStatusAsync(Guid.Parse(textAnalyticsJob.Id)).ConfigureAwait(false);
+
+                if (response.IsError)
+                {
+                    continue;
+                }
+
+                var analysisResult = JsonConvert.DeserializeObject<AnalyzeConversationsResult>(response.Content.ToString());
+
+                if (analysisResult.Tasks.InProgress != 0)
+                {
+                    // some jobs are still running.
+                    runningJobsCount++;
+                }
+            }
+
+            return runningJobsCount == 0;
+        }
+
+        private async Task<(IEnumerable<string> jobId, IEnumerable<string> errors)> SubmitConversationsAsync(dynamic data)
+        {
+            var errors = new List<string>();
+
+            try
+            {
+                Log.LogInformation($"Sending language conversation request.");
+
+                var operation = await ConversationAnalysisClient.AnalyzeConversationAsync(WaitUntil.Started, RequestContent.Create(JsonConvert.SerializeObject(data))).ConfigureAwait(false);
+
+                var response = await operation.UpdateStatusAsync().ConfigureAwait(false);
+
+                using JsonDocument result = JsonDocument.Parse(response.ContentStream);
+                var jobResults = result.RootElement;
+                var jobId = jobResults.GetProperty("jobId");
+
+                return (new List<string> { jobId.GetString() }, errors);
+            }
+            catch (OperationCanceledException)
+            {
+                throw new TimeoutException($"The operation has timed out after {RequestTimeout.TotalSeconds} seconds.");
+            }
+
```

Comment:
remove new line here

Reply:
there is comment below so need the new line here :-)